### ISSUE TITLE
Circuit entity

### ISSUE THREAD

**Issue by rosscado** [rosscado/raceready#43]
Create a circuit entity at endpoint `/api/circuits/`

https://github.com/rosscado/raceready/wiki/Data-Model#circuit

*Schema*
```yaml
id:
URL: web address of the mapped route or segment, e.g. Strava route/segment
elevation_gain_m: elevation gained (in metres) over one lap of the circuit, e.g. 250
distance_km: length (in kilometres) of one lap of the circuit, e.g. 12
flatness_index: computed, indicates how close to flat is the circuit, e.g. 0.15
```

*Example*
```yaml
  - name: Create the circuit
    request:
      url: "{api.server}/api/circuits/"
      json:
        url: https://www.mapmyride.com/routes/view/2189856949
        distance_km: 86.95
        elevation_gain_m: 1315
      method: POST
      headers:
        content-type: application/json
    response:
      status_code: 201
```

For now all fields except `flatness_index` will be populated by client POST/PUT requests exclusively. In later iterations `distance_km` and `elevation_gain_m` and other fields may be parsed from the route information at `url`.

**Comment by rosscado** [comment]
Closed by 3b62e3cda7bead70eda265f140b5506a9424c951

**Comment by rosscado** [comment]
IBM Cloud [toolchain](https://console.eu-gb.bluemix.net/devops/toolchains/0fc11092-0119-4ab8-a5f0-6cbf19d20e03?env_id=ibm%3Ayp%3Aeu-gb "raceready"): [Delivery Pipeline](https://console.eu-gb.bluemix.net/devops/pipelines/8de9b026-388d-4ebc-984e-5928d63a1d84?env_id=ibm%3Ayp%3Aeu-gb "drivetrain") deployed [raceready](https://github.com/rosscado/raceready/releases/tag/deploy-ibm-yp-eu-gb-VeloWare-staging-20180906-145823 "deploy-ibm-yp-eu-gb-VeloWare-staging-20180906-145823") to [staging](https://console.bluemix.net/apps/23d83a9d-7943-46e3-b4ec-d36604aa66e2?env_id=ibm:yp:eu-gb "ibm:yp:eu-gb:VeloWare:staging"), including commit 2982655daeedde12c91a7c8e5bdd57e9cc8d2e61
